tests/utils.py

```diff
@@ -284,7 +284,7 @@ def __init__(self, lightning_dir, bitcoin_dir, port=9735, random_hsm=False):
             self.opts['dev-broadcast-interval'] = 1000
             if not random_hsm:
                 self.opts['dev-hsm-seed'] = binascii.hexlify(seed).decode('ascii')
-        self.prefix = 'lightningd(%d)' % (port)
+        self.prefix = 'lightningd-%d' % (node_id)
```

Comment:
If two tests run in parallel, would they get separate `node_factory`?  We instantiate a new factory at `setUp`.  Sorry, but I am unfamiliar with the Python test environment (and Python in general); can tests in a single test class run in parallel or do they have to be separate standalone functions like in `test_gossip.py` in order to run in parallel?